gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/portal_page/domain_service/PortalNavigationItemValidatorService.java

```diff
@@ -106,4 +111,36 @@ private boolean isValidUrl(String url) {
             return false;
         }
     }
+
+    public void validateToUpdate(UpdatePortalNavigationItem toUpdate, PortalNavigationItem existingItem) {
+        enforceTypeConsistency(existingItem, toUpdate.getType());
+
+        if (toUpdate.getTitle() == null || toUpdate.getTitle().isBlank()) {
```

Comment:
Null is handled at the spec level
```suggestion
        if (toUpdate.getTitle().isBlank()) {